pages/about/index.html

```diff
@@ -0,0 +1,15 @@
+<!-- See structure.html for the template names used here and their position in the layout. -->
+{{define "title"}}Team Silverblue &mdash; About{{end}}
+
+{{define "body"}}
+
+<article>
+<h1>What is Silverblue?</h1>
+<p>
+  Silverblue is a new generation of the desktop operating system formerly known as Atomic Workstation.
```

Comment:
Can you update that with the intro text from https://docs.fedoraproject.org/en-US/fedora-silverblue/ ?